### PR TITLE
Update Claims and Assets pages

### PR DESCRIPTION
**Pull request by sibayanjasper**
### a. Implementation Changes:
- Remove CMS column in Claims page
- Update CMS column drop-down after switching to different music provider

### a. Ticket link/Request Feature link
- [Dev-827](https://freedom.myjetbrains.com/youtrack/issue/Dev-827)

#### Assets page
- [ ] 1. Go to http://10.10.13.50:8080/#/music-provider/positionmusic/music-assets.
- [ ] 2. CMS column should show all CMSs associated with Position Music:
  - Freedom! CMS
  - Position Music Publishing CMS
  - Position Music Sound Recording CMS
  - Unknown CMS
- [ ] 3. Select TrackTesting from the Use as Music Provider menu.
- [ ] 4. CMS column should only show CMSs associated with TrackTesting:
  - Freedom! CMS
- [ ] 5. CMSs should be sorted.

#### Claims page
- [ ] 1. Go to http://10.10.13.50:8080/#/music-provider/tracktesting/claims.
- [ ] 2. CMS column should now be Asset CMS.
- [ ] 3. Filtering by Asset CMS should only show claims associated with the selected asset CMS.
- [ ] 4. Go to http://10.10.13.50:8080/#/admin/disputed-claims.
- [ ] 5. CMS column should now be Asset CMS.
- [ ] 6. Asset CMSs should be sorted.
- [ ] 7. Filtering by Asset CMS should only show the disputed claims associated with the selected asset CMS.

### CODE REVIEW

File: app/scripts/components/claims/disputed.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ var React = require('react'),
                 total: DisputesStore.getTotal(),
                 results: results,
                 filter: '',
-                columns: ['video_views', 'video_title', 'channel_title', 'asset_title', 'reason', 'cms', 'expiration', 'claim_id'],
+                columns: ['video_views', 'video_title', 'channel_title', 'asset_title', 'reason', 'expiration', 'claim_id'],
```
Comment (wilbertverayin) [inline app/scripts/components/claims/disputed.js]: line is long. please split

File: app/scripts/components/claims/index.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ var React = require('react'),
                 total: ClaimsStore.getTotal(),
                 results: ClaimsStore.getClaims(),
                 filter: '',
-                columns: ['video_views', 'video_title', 'channel_title', 'asset_title', 'cms', 'insert_date', 'update_date', 'release_date', 'status', 'claim_id'],
+                columns: ['video_views', 'video_title', 'channel_title', 'asset_title', 'insert_date', 'update_date', 'release_date', 'status', 'claim_id'],
```
Comment (wilbertverayin) [inline app/scripts/components/claims/index.js]: line is long please split

File: app/scripts/components/claims/disputed.js (hunk 1 above)
Comment (sibayanjasper) [inline app/scripts/components/claims/disputed.js]: _updated_

File: app/scripts/components/claims/index.js (hunk 2 above)
Comment (sibayanjasper) [inline app/scripts/components/claims/index.js]: _updated_

**Comment by wilbertverayin** [comment]
#### Assets page
- [ ] 1. Go to http://10.10.13.50:8080/#/music-provider/positionmusic/music-assets.
- [ ] 2. CMS column should show all CMSs associated with Position Music:
  - Freedom! CMS
  - Position Music Sound Recording CMS
  - Position Music Publishing CMS
  - Unknown CMS
- [ ] 3. Select TrackTesting from the Use as Music Provider menu.
- [ ] 4. CMS column should only show CMSs associated with TrackTesting:
  - Freedom! CMS

#### Claims page
- [ ] 1. Go to http://10.10.13.50:8080/#/music-provider/tracktesting/claims.
- [ ] 2. CMS column should now be removed.
- [ ] 3. Go to http://10.10.13.50:8080/#/music-provider/tracktesting/disputed-claims.
- [ ] 4. CMS column should now be removed.

**Comment by sibayanjasper** [comment]
@wilbertverayin, I updated the test cases. Please test again.

**Comment by wilbertverayin** [comment]
#### Assets page
- [ ] 1. Go to http://10.10.13.50:8080/#/music-provider/positionmusic/music-assets.
- [ ] 2. CMS column should show all CMSs associated with Position Music:
  - Freedom! CMS
  - Position Music Publishing CMS
  - Position Music Sound Recording CMS
  - Unknown CMS
- [ ] 3. Select TrackTesting from the Use as Music Provider menu.
- [ ] 4. CMS column should only show CMSs associated with TrackTesting:
  - Freedom! CMS
- [ ] 5. CMSs should be sorted.

#### Claims page
- [ ] 1. Go to http://10.10.13.50:8080/#/music-provider/tracktesting/claims.
- [ ] 2. CMS column should now be Asset CMS.
- [ ] 3. Filtering by Asset CMS should only show claims associated with the selected asset CMS.
- [ ] 4. Go to http://10.10.13.50:8080/#/admin/disputed-claims.
- [ ] 5. CMS column should now be Asset CMS.
- [ ] 6. Asset CMSs should be sorted.
- [ ] 7. Filtering by Asset CMS should only show the disputed claims associated with the selected asset CMS.

**Pull request merged**